src/Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/BjorhusImpl.cpp

```diff
@@ -0,0 +1,658 @@
+        char_projected_rhs_dt_v_psi) noexcept {
+  // gauge_bc_coeff below is hard-coded here to its default value in SpEC
+  constexpr double gauge_bc_coeff = 1.;
+
+  DataType inertial_radius_or_scalar_factor(get_size(get<0>(inertial_coords)),
+                                            0.);
+  for (size_t i = 0; i < VolumeDim; ++i) {
+    inertial_radius_or_scalar_factor += square(inertial_coords.get(i));
+  }
+  inertial_radius_or_scalar_factor =
+      get(gamma2) -
+      gauge_bc_coeff * (1. / sqrt(inertial_radius_or_scalar_factor));
+
+  // add in gauge BC
+  for (size_t a = 0; a <= VolumeDim; ++a) {
+    for (size_t b = a; b <= VolumeDim; ++b) {
+      for (size_t c = 0; c <= VolumeDim; ++c) {
+        for (size_t d = 0; d <= VolumeDim; ++d) {
+          bc_dt_v_minus->get(a, b) +=
+              0.5 *
+              (incoming_null_one_form.get(a) * projection_Ab.get(c, b) *
+                   outgoing_null_vector.get(d) +
+               incoming_null_one_form.get(b) * projection_Ab.get(c, a) *
+                   outgoing_null_vector.get(d) -
+               0.5 * (incoming_null_one_form.get(a) *
+                          outgoing_null_one_form.get(b) *
+                          incoming_null_vector.get(c) *
+                          outgoing_null_vector.get(d) +
+                      incoming_null_one_form.get(b) *
+                          outgoing_null_one_form.get(a) *
+                          incoming_null_vector.get(c) *
+                          outgoing_null_vector.get(d) +
+                      incoming_null_one_form.get(a) *
+                          incoming_null_one_form.get(b) *
+                          outgoing_null_vector.get(c) *
+                          outgoing_null_vector.get(d))) *
+              inertial_radius_or_scalar_factor *
+              char_projected_rhs_dt_v_psi.get(c, d);
+        }
+      }
+    }
+  }
+}
+
+template <size_t VolumeDim, typename DataType>
+void add_constraint_dependent_terms_to_dt_v_minus(
+    const gsl::not_null<tnsr::aa<DataType, VolumeDim, Frame::Inertial>*>
+        bc_dt_v_minus,
+    const tnsr::a<DataType, VolumeDim, Frame::Inertial>& outgoing_null_one_form,
+    const tnsr::A<DataType, VolumeDim, Frame::Inertial>& incoming_null_vector,
+    const tnsr::A<DataType, VolumeDim, Frame::Inertial>& outgoing_null_vector,
+    const tnsr::aa<DataType, VolumeDim, Frame::Inertial>& projection_ab,
+    const tnsr::Ab<DataType, VolumeDim, Frame::Inertial>& projection_Ab,
+    const tnsr::AA<DataType, VolumeDim, Frame::Inertial>& projection_AB,
+    const tnsr::a<DataType, VolumeDim, Frame::Inertial>&
+        constraint_char_zero_plus,
+    const tnsr::a<DataType, VolumeDim, Frame::Inertial>&
+        constraint_char_zero_minus,
+    const tnsr::aa<DataType, VolumeDim, Frame::Inertial>&
+        char_projected_rhs_dt_v_minus,
+    const std::array<DataType, 4>& char_speeds) noexcept {
+  constexpr double mu = 0.;  // hard-coded value from SpEC Bbh input file Mu = 0
```

Comment:
From discussions with Mark and Larry we can just omit this term. If you really want to keep it could you hide it behind an `if constexpr` so we don't do the extra work of computing something that is always identically zero?

Reply:
I think this still needs to be done

Reply:
I've moved all terms prop to `mu` behind an `if constexpr`.